### PR TITLE
Fixes to pom property versions and readme files

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

**Comment by sgilda** [comment]
Good catch Shelly! Made the change, squashed the commits.

File: cluster-ha-singleton/README.md [hunk 1]
```diff
@@ -58,9 +58,9 @@ _NOTE: The following build command assumes you have configured your Maven user s
         mvn clean install jboss-as:deploy
 
 4. This will deploy `service/target/jboss-as-cluster-ha-singleton-service.jar` to the running instance of the server.
-5. Type this command to deploy the archive to the second server (or more) and replace hostname and port depend on your settings:
+5. Type this command to deploy the archive to the second server (or more) and replace the port, depending on your settings:
 
-        mvn jboss-as:deploy -Ddeploy.hostname=localhost -Ddeploy.port=10099
+        mvn jboss-as:deploy -Ddeploy.port=10099
```
Comment (rafabene) [inline cluster-ha-singleton/README.md]: The property name for 7.3 is jboss-as.deploy

File: cluster-ha-singleton/README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Undeploy the Archive
 3. When you are finished testing, type this command to undeploy the archive:
 
         mvn jboss-as:undeploy
-        mvn jboss-as:undeploy -Ddeploy.hostname=localhost -Ddeploy.port=10099
```
Comment (rafabene) [inline cluster-ha-singleton/README.md:107]: The property name for 7.3 is jboss-as.deploy

File: ejb-asynchronous/README.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ Undeploy the Archive
 3. When you are finished testing, type this command to undeploy the archive:
 
         mvn jboss-as:undeploy
-        mvn jboss-as:undeploy -Ddeploy.hostname=localhost -Ddeploy.port=10099
+        mvn jboss-as:undeploy -Ddeploy.port=10099
```
Comment (rafabene) [inline ejb-asynchronous/README.md]: The property name for 7.3 is jboss-as.deploy

**Comment by rafabene** [comment]
It seems ok now!

**Comment by sgilda** [comment]
Merged.

**Pull request closed without merging**